cmd/install/assets/hypershift_operator.go

```diff
@@ -558,3 +580,20 @@ func (r HypershiftRecordingRule) Build() *prometheusoperatorv1.PrometheusRule {
 	rule.Spec = recordingRuleSpec()
 	return rule
 }
+
+func AWSOIDCBucketConfigMap(bucketName, bucketRegion string) *corev1.ConfigMap {
```

Comment:
Did I get it right this is putting all clusters config into the management cluster `kube-public` ns?

Reply:
No, it is a configmap that contains the name and the region of the bucket that contains the OIDC docs. `install iam` needs this to be able to set up the OIDC provider